lib/literal/properties.rb

```diff
@@ -6,8 +6,13 @@ module Literal::Properties
 
 	include Literal::Types
 
+	module DocString
+		# @!method initialize(...)
+	end
+
 	def self.extended(base)
 		super
+		base.include(DocString)
```

Comment:
Can RubyMine follow dynamic includes on the `extended` callback?

Reply:
Yeah this works, RubyMine seems to understand the include here.

Reply:
If it can follow this, could it follow a `prepend` with a real initialize?

Reply:
Possibly, just wondering if a solution that adds as little real code as possible is nicer. Eg if the issue only really bothers RubyMine users, they could either add that doctstring to a mixin/baseclass etc themselves, or we could provide a script to generate docstrings or RBS for literal props that could then be added to ones project (which would also be more useful as the IDE would know the real signature of the initialize rather than a catch-all)

Reply:
Makes sense.